src/App.jsx

```diff
@@ -1,25 +1,36 @@
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import { ThemeProvider } from 'styled-components';
 
 import './styles/common/reset.css';
 import './styles/common/variables.css';
 import Content from './router/Content';
 import { light, dark } from './styles/themes/theme';
 import GlobalStyles from './styles/common/global.styles';
-import { ToastContainer } from 'react-toastify';
+import { StyledToastContainer } from './components/toast/Toast.Styles'; // StyledToastContainer 가져오기
 
 function App() {
   const [isDarkMode, setIsDarkMode] = useState(false);
   const theme = isDarkMode ? dark : light;
 
   const toggleTheme = () => {
-    setIsDarkMode((prevMode) => !prevMode);
+    setIsDarkMode((prevMode) => {
+      const isMode = !prevMode;
+      localStorage.setItem('mode', isMode ? 'dark' : 'light');
```

Comment:
여기가 로컬스토리지 이용한 부분인가요??
생각보다 너무 간단하네요..ㅋㅋ 혹시 로컬스토리지에 담겼는지 확인은 어떻게 하는지 알려주실수있나요??
```suggestion
      localStorage.setItem('mode', isMode ? 'dark' : 'light');
```

Reply:
<img width="532" alt="스크린샷 2024-11-06 22 35 19" src="https://github.com/user-attachments/assets/91fe6937-e943-498f-93b7-93c96eae89d3">
개발자모드드의 애플리케이션에서 확인하실 수 있습니다!

Reply:
오오 좋네용!! 확인 감사합니당 ㅎㅎ